app/views/users/register.haml

```diff
@@ -20,6 +20,11 @@
       = f.label :first_time_attendee, 'Will this be your first time at beCamp?'
       = f.select :first_time_attendee, options_for_select([['Please choose one...', ''], ['Yes', true], ['No', false]], current_user.first_time_attendee)
 
+    = field_set_tag do
+      = f.label :list_me, 'Can we show your name?'
+      = f.radio_button :list_me, "True", :checked => true
```

Comment:
I agree that the visibility has value for creating interest. I think it also has the potential to turn people away. My preference would be a post-registration prompt to make yourself visible in the list. Maybe land the user on a social/sharing page with options to become visible, tweet, post to PlusBook, etc.